python/tests/integration/arcticdb/test_update.py

```
@@ -555,3 +557,21 @@ def test_update_batch_different_updates_dynamic_schema(custom_library):
         ArcticSymbolSimulator.assert_frame_equal_rebuild_index_first(
             expected_results[result.symbol], read_data[result.symbol].data
         )
+
+
+@pytest.mark.parametrize(
+    "idx", [pd.date_range(pd.Timestamp("2020-01-01"), periods=3), pd.RangeIndex(start=0, stop=3, step=1)]
```

nit: can't update with a range index anyway

---

Yeah but the normalization path is a bit different for the 2 index types so I prefer to keep this check.
If we do manage to normalize the index, it should raise a different exception, so the test won't pass in that case.